src/Tizen.NUI/src/public/WindowSystem/QuickPanelClient.cs

```diff
@@ -0,0 +1,386 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Tizen.Applications.Exceptions;
+
+namespace Tizen.NUI
+{
+    namespace WindowSystem
+    {
+        namespace WsShell
+        {
+            /// <summary>
+            /// Class for the Tizen quickpanel client.
+            /// </summary>
+            public class QuickPanelClient : IDisposable
+            {
+                private TzShell _tzsh;
+                private IntPtr _tzshQpClient;
+                private int _tzshWin;
+                private bool disposed = false;
+
+                private int _visibleEventType;
+                private IntPtr _visibleEventHandler;
+                private Interop.QuickPanelClient.QuickPanelEventCallback _OnVisibleChanged;
+                private event EventHandler<StateVisible> _VisibleEventHandler;
```

Comment:
wrong event handler name: https://docs.microsoft.com/ko-kr/dotnet/standard/design-guidelines/names-of-type-members#names-of-events
and start with lower character

Reply:
I changed _VisibleEventHadler -> _visibleChanged